public/app/features/provisioning/api/endpoints.ts

```diff
@@ -218,6 +224,15 @@ const injectedRtkApi = api
         }),
         invalidatesTags: ['Repository'],
       }),
+      getApisProvisioningGrafanaAppV0Alpha1NamespacesByNamespaceStats: build.query<
```

Comment:
Do we need this endpoint or is this data also available from the repository list endpoint? We can filter it out if not needed.

Reply:
that is a new endpoint that gets the stats for everything at once -- I set a better operaionId for it, and it now looks ok